src/thriveopedia/pages/ThriveopediaMuseumPage.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Linq;
+using Godot;
+
+/// <summary>
+///   Thriveopedia page displaying fossilised (saved) organisms.
+/// </summary>
+public class ThriveopediaMuseumPage : ThriveopediaPage
+{
+    [Export]
+    public NodePath CardContainerPath = null!;
+
+    [Export]
+    public NodePath WelcomeLabelPath = null!;
+
+    [Export]
+    public NodePath SpeciesPreviewContainerPath = null!;
+
+    [Export]
+    public NodePath SpeciesPreviewPath = null!;
+
+    [Export]
+    public NodePath HexesPreviewPath = null!;
+
+    [Export]
+    public NodePath SpeciesDetailsLabelPath = null!;
+
+    [Export]
+    public NodePath LeaveGameConfirmationDialogPath = null!;
+
+    private GridContainer cardContainer = null!;
+    private Control welcomeLabel = null!;
+    private VBoxContainer speciesPreviewContainer = null!;
+    private SpeciesPreview speciesPreview = null!;
+    private CellHexesPreview hexesPreview = null!;
+    private CustomRichTextLabel speciesDetailsLabel = null!;
+    private CustomConfirmationDialog leaveGameConfirmationDialog = null!;
+
+    public override string PageName => "Museum";
+    public override string TranslatedPageName => TranslationServer.Translate("MUSEUM_PAGE");
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        cardContainer = GetNode<GridContainer>(CardContainerPath);
+        welcomeLabel = GetNode<Control>(WelcomeLabelPath);
+        speciesPreviewContainer = GetNode<VBoxContainer>(SpeciesPreviewContainerPath);
+        speciesPreview = GetNode<SpeciesPreview>(SpeciesPreviewPath);
+        hexesPreview = GetNode<CellHexesPreview>(HexesPreviewPath);
+        speciesDetailsLabel = GetNode<CustomRichTextLabel>(SpeciesDetailsLabelPath);
+        leaveGameConfirmationDialog = GetNode<CustomConfirmationDialog>(LeaveGameConfirmationDialogPath);
+    }
+
+    public override void _Notification(int what)
+    {
+        base._Notification(what);
+
+        if (what == NotificationVisibilityChanged && Visible)
+        {
+            // For now, rebuild the card list entirely each time we open the page. Very unoptimised, but it keeps
+            // the museum up to date with the player's new fossilisations in a game. A possible next step would be
+            // to rebuild only when the Thriveopedia as a whole is opened.
+            cardContainer.QueueFreeChildren();
+
+            foreach (var speciesName in FossilisedSpecies.CreateListOfSaves())
+            {
+                var card = (MuseumCard)GD.Load<PackedScene>(
+                    "res://src/thriveopedia/fossilisation/MuseumCard.tscn").Instance();
+
+                var savedSpecies = FossilisedSpecies.LoadSpeciesFromFile(speciesName);
+
+                if (savedSpecies is not MicrobeSpecies)
+                {
+                    throw new NotImplementedException("Loading non-microbe species is not yet implemented");
+                }
```

Comment:
I think this throw is a bit much here, I think the card should just reduce the available functions it has if given another type of species.